ModelBundle/Repository/ContentRepository.php

```diff
@@ -169,12 +182,12 @@ protected function createQueryWithLanguage($language = null)
 
     /**
      * @param string      $contentId
-     * @param string|null $language
+     * @param string      $language
      * @param int|null    $version
      *
      * @return Builder
      */
-    protected function createQueryWithDefaultCriteria($contentId, $language = null, $version = null)
+    protected function createQueryWithDefaultCriteria($contentId, $language, $version = null)
```

Comment:
Maybe this should be renamed : there is no more default criteria as they are explicitly given to the method.